packages/republik-crowdfundings/lib/Mail.js

```diff
@@ -673,6 +673,60 @@ mail.sendMembershipClaimNotice = async ({ membership }, { pgdb, t }) => {
   )
 }
 
+mail.sendMembershipClaimerOnboarding = async (
+  { claimedMembership, activeMembership },
+  { pgdb, t },
+) => {
+  const claimer = await pgdb.public.users.findOne({
+    id: claimedMembership.userId,
+  })
+
+  const mailTemplateData = {
+    to: claimer.email,
+    fromEmail: process.env.DEFAULT_MAIL_FROM_ADDRESS,
+    templateName: 'membership_claimer_onboarding',
+    mergeLanguage: 'handlebars',
+    subject: t(`api/email/membership_claimer_onboarding/DEFAULT/subject`),
+  }
+
+  const claimedMembershipCompany = await pgdb.queryOneField(`
+    SELECT c.name
+    FROM "membershipTypes" mt
+    JOIN companies c
+      ON mt."companyId" = c.id
+    WHERE mt.id = '${claimedMembership.membershipTypeId}'
+    LIMIT 1
+  `)
+
+  if (activeMembership) {
+    const activeMembershipCompany = await pgdb.queryOneField(`
+      SELECT c.name
+      FROM "membershipTypes" mt
+      JOIN companies c
+        ON mt."companyId" = c.id
+      WHERE mt.id = '${activeMembership.membershipTypeId}'
+      LIMIT 1
+    `)
+
+    mailTemplateData.subject = t(
+      `api/email/membership_claimer_onboarding/ACTIVE/subject`,
+    )
+
+    mailTemplateData.globalMergeVars = [
+      {
+        name: 'active_membership_type_company_name',
+        content: activeMembershipCompany,
+      },
+    ]
+  } else if (claimedMembershipCompany) {
+    mailTemplateData.subject = t(
+      `api/email/membership_claimer_onboarding/${claimedMembershipCompany}/subject`,
+    )
+  }
+
+  return sendMailTemplate(mailTemplateData, { pgdb })
```

Comment:
Here is an (mock) attempt to reduce LOC and cognitive load.

`t.first` accepts an array of translation keys and uses the first present. This is allows us to group possibilites at glance.

```js
const keys = [
  hasVegetables && `burger/with_vegetables/${type}/subject`,
  hasVegetables && `burger/with_vegetables/subject`,
  `burger/${type}/subject`,
  `burger/subject`,
].filter(Boolean)

t.first(keys)
```

That is a pattern we do cherish.

Using this, shovel redundat query into a util function and stripping some if-else-conditionals,`sendMembershipClaimerOnboarding` could look like this: 

```js
  // Put this near to of lib/Mail.js
  const getMembershipTypeCompany = (membership) => {
    const { membershipTypeId } = membership

    return pgdb.queryOneField(`
      SELECT c.name
      FROM "membershipTypes" mt
      JOIN companies c
        ON mt."companyId" = c.id
      WHERE mt.id = '${membershipTypeId}'
      LIMIT 1
    `)
  }

  // in sendMembershipClaimerOnboarding then

  const claimer = await pgdb.public.users.findOne({
    id: claimedMembership.userId,
  })
  const claimedMembershipCompany = await getMembershipTypeCompany(
    claimedMembership,
  )
  const activeMembershipCompany =
    activeMembership && (await getMembershipTypeCompany(activeMembership))

  return sendMailTemplate(
    {
      to: claimer.email,
      fromEmail: process.env.DEFAULT_MAIL_FROM_ADDRESS,
      templateName: 'membership_claimer_onboarding',
      mergeLanguage: 'handlebars',
      subject: t.first(
        [
          activeMembership &&
            `api/email/membership_claimer_onboarding/has_active_membership/subject`, // New key, instead ACTIVE
          `api/email/membership_claimer_onboarding/${claimedMembershipCompany}/subject`,
          `api/email/membership_claimer_onboarding/subject`, // New key, DEFAULT removed
        ].filter(Boolean),
      ),
      globalMergeVars: [
        {
          name: 'active_membership_type_company_name',
          content: activeMembershipCompany, // Works if falsy
        },
      ],
    },
    { pgdb },
  )
```